src/styles/index.css

```diff
@@ -0,0 +1,38 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  gap: 10px;
+  width: 80%;
+  max-width: 600px;
+}
+
+.day {
```

Comment:
The class name `.day` does not match the class name `calendar__day` used in the HTML file. Ensure that the class names are consistent across your HTML and CSS files to apply the styles correctly.